web/styles/typeahead.css

```diff
@@ -176,20 +177,20 @@
 
 .typeahead-image {
     display: inline-block;
-    height: 21px;
-    width: 21px;
+    height: 1.1052em; /* 21px at 19px font size at 14px em */
+    width: 1.1052em; /* 21px at 19px font size at 14px em */
     border-radius: 4px;
 
     /* For FontAwesome icons and zulip icons used in place of images for some users. */
-    font-size: 19px;
+    font-size: 1.3571em; /* 19px at 14px em */
     text-align: center;
 
     &.zulip-icon-triple-users {
-        font-size: 19px;
+        font-size: 1.3571em; /* 19px at 14px em */
     }
 
     &.no-presence-circle {
-        margin-left: 14px;
+        margin-left: 0.7368em; /* 14px at 19px font size at 14px em */
```

Comment:
I'm having a hard time following this.

Reply:
Updated to use the "at 19px/1em" you suggested above